core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.html

```diff
@@ -44,15 +44,15 @@
                alt="Save Draft">
               &#xE161;
             </i>
-            <span class="oppia-save-draft-button-label oppia-save-publish-button-label">Save Draft</span>
+            <span class="oppia-save-draft-button-label oppia-save-publish-button-label protractor-test-save-draft">Save Draft</span>
```

Comment:
Instead of `protractor-test-save-draft`, maybe use `protractor-test-save-draft-message`? I think this gives a better idea of what the element actually is

Reply:
Done